src/routes/Profile/Competences/constants.ts

```diff
@@ -12,6 +12,7 @@ export interface FilterData {
 }
 
 export type FiltersType = Partial<Record<OsaaminenLahdeTyyppi, FilterData[]>>;
+export const desiredFilterKeyOrder = ['TOIMENKUVA', 'KOULUTUS', 'PATEVYYS', 'KIINNOSTUS', 'MUU_OSAAMINEN'];
```

Comment:
I'd change the variable name to `FILTERS_ORDER` and add `as const` so it becomes read only type:

`export const FILTERS_ORDER = [...] as const`